result/dhcpv4server.yaml

```diff
@@ -0,0 +1,113 @@
+openapi: 3.0.3
+info:
+  title: DHCPv4 server results models
+  version: ^0.0.0
+components:
+  schemas:
+    Dhcpv4Server.Metrics.Request:
+      description: >-
+        The request to retrieve DHCPv4 per Server metrics/statistics.
+      type: object
+      properties:
+        server_names:
+          description: >-
+            The names of DHCPv4 Servers to return results for.
+            An empty list will return results for all DHCPv4 Server.
+          type: array
+          items:
+            type: string
+          x-constraint:
+          - "/components/schemas/Device.Dhcpv4Server/properties/name"
+          x-field-uid: 1
+        column_names:
+          description: >-
+            The list of column names that the returned result set will contain.
+            If the list is empty then all columns will be returned except for
+            any result_groups. The name of the DHCPv4 server cannot be excluded.
+          type: array
+          items:
+            type: string
+            x-enum:
+              session_up:
+                x-field-uid: 1
+              session_down:
+                x-field-uid: 2
+              discovers_sent:
+                x-field-uid: 3
+              offers_received:
+                x-field-uid: 4
+              requests_sent:
+                x-field-uid: 5
+              acks_received:
+                x-field-uid: 6
+              nacks_received:
+                x-field-uid: 7
+              releases_sent:
+                x-field-uid: 8
+              declineds_sent:
+                x-field-uid: 7
+          x-field-uid: 2
+    Dhcpv4Server.Metric:
+      description: >-
+        DHCPv4 per peer statistics information.
+      type: object
+      properties:
+        name:
+          description: >-
+            The name of a configured DHCPv4 Server.
+          type: string
+          x-field-uid: 1
+        session_up_count:
+          description: >-
+            The number of sessions that are up.
+          type: integer
+          format: uint64
+          x-field-uid: 2
+        session_down_count:
+          description: >-
+            The number of sessions that are down.
```

Comment:
How will we know this ... are all clients pre-configured ?

Reply:
Client session count put in down state when the dhcpdiscovery message is received and that becomes up when server sends discpack message

Reply:
Will these two events happen with a time gap asynchronously ? Else down count will always be 0 from user point of view ?
Also say some clients are configured incorrectly and not sending requests correctly or requests being rejected .. then up + Down will not be equal to configured . Thinking whether that might cause confusion.